### PR TITLE
Sort/Pagination integrated into series layout; Journal landing improvements; NotYet style fix

### PR DESCRIPTION
**Pull request by amardesi**

### CODE REVIEW

File: app/server.rb [hunk 1]
```diff
@@ -498,8 +498,13 @@ def getChildDepts(unit)
...
       sidebar: getUnitSidebar(unit)
     }
-    if pageName == 'home'
-      pageData[:content] = getUnitPageContent(unit, attrs)
+    if ["home", "search"].include? pageName
+      q = nil
+      q = CGI::parse(request.query_string) if pageName == "search"
+      pageData[:content] = getUnitPageContent(unit, attrs, q)
+    # This is subsumed under getUnitPageContent right now
+    # elsif pageName == 'search'
```
Comment (amardesi) [inline app/server.rb]: probably should remain as such?

File: app/jsx/layouts/JournalLayout.jsx [hunk 2]
```diff
@@ -67,23 +144,18 @@ class JournalLayout extends React.Component {
   }
   
   render() {
-    var data = this.props.data;
-
-    var seriesList = [];
```
Comment (amardesi) [inline app/jsx/layouts/JournalLayout.jsx]: Unnecessary code most likely duplicate from DepartmentLayout. Not needed here.

File: app/jsx/components/HeatMapComp.jsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ class HeatMapComp extends React.Component {
     return (
       <div className="c-heatmap" style={{backgroundImage: "url('/images/sample_heatmap.png')"}}>
         <h1 className="c-heatmap__heading">Share your research with a <br/><strong>global audience</strong></h1>
-        <NotYetLink><button className="c-heatmap__button">Get Started</button></NotYetLink>
+        <NotYetLink className="c-heatmap__button" element="button">Get Started</NotYetLink>
```
Comment (martinhaye) [inline app/jsx/components/HeatMapComp.jsx]: Tested, and it works nicely!

File: app/server.rb (hunk 1 above)
Comment (martinhaye) [inline app/server.rb]: Sure, until we need to change it.

**Pull request merged**